deploy.sh

```diff
@@ -1,5 +1,35 @@
 #!/bin/bash
 
+# variables used for logging build steps, use build_step() and
+# build_step_done() for setting or clearing them.
+BUILD_STEP_FILE="$(mktemp)"
```

Comment:
one better way to test these changes is remove travis_wait in https://github.com/ceph/ceph-csi/blob/master/.travis.yml#L113 and move this code to a new file or function which can be used to build image also

Reply:
Right, removed the `travis_wait` command. Also added a new commit to test the deploy script in a separate job (the job is marked to fail, need to check the logs).